### DISCUSSION TITLE
Please help me to add some of your favorite Japanese foods into my list.

### DISCUSSION THREAD

**Discussion by eedy8888** [NAIST-SE/AttractingContributors#67] [General]
The url is below. Thanks.

https://github.com/eedy8888/AttractingContributors

**Comment by mailtoy** [comment]
added please help me https://github.com/mailtoy/AttractingContributors-Movies

**Reply by eedy8888** [reply]
Thank you, I added

**Comment by 278Mt** [comment]
added!

**Reply by eedy8888** [reply]
Thanks

**Comment by manzoku23** [comment]
Added my one.
Please help me 
https://github.com/Exe-dev/AttractingContributors-POKEMON

**Reply by eedy8888** [reply]
Thank you, I added

**Reply by manzoku23** [reply]
thx :D

**Comment by sergiodl-naist** [comment]
Added one more!
Please visit and try to ask a question or recommend one vtuber on my fork!
https://github.com/sergiodl-naist/AttractingContributors-VTubers
